README.md

```diff
@@ -1,3 +1,99 @@
+
+The build is minified and the filenames include the hashes.<br>
+Your app is ready to be deployed!
+
+See the section about [deployment](#deployment) for more information.
+
+### `npm run lint`
+
+Runs lint on the project sources.
+
+### `npm run lint:fix`
+
+Runs lint on the project sources with the `--fix` option, which automatically corrects problems (like missing semicolons) that can be fixed programmatically. Any modified source files will need to be committed manually.
+
+## `Populations.js` Wrapper
+
+The [Populations.js](https://github.com/concord-consortium/populations.js) library was developed in [CoffeeScript](https://coffeescript.org/) and bundled with [brunch](http://brunch.io/). Historically, models built using Populations.js were often developed with CoffeeScript and brunch as well, often as part of the [Populations Models](https://github.com/concord-consortium/populations-models) project. For this project we developed a wrapper for the Populations.js library that allows it to be used from a TypeScript application bundled with WebPack. TypeScript definitions at [populations.d.ts](src/populations.d.ts) provide a typed interface to the capabilities of the Populations.js library.
+
+A pre-built version of the Populations.js library is included in the `public` folder with the name `vendor.js`. A local JavaScript file [populations.js](src/populations.js) imports the required symbols from `vendor.js` using the `require_brunch` function and then exports them for consumption by the rest of the application. For this to work, the symbol `require_brunch` is configured as an alias for `require` at runtime in `index.html`. (`require_brunch` is used to prevent WebPack from processing it like regular `require`s). For unit testing, a hacked version of the Populations.js library is available in the `test` folder as `vendor-test.js` in which unused libraries were manually removed (to improve load times) and the `require_brunch` symbol is explicitly defined as an alias for require. The `vendor-test.js` library is then loaded into JSDom in [setupTests.ts](src/setupTests.ts).
+
+Updating to a different version of the Populations.js library would require replacing `vendor.js` with the new version, and then producing a hacked version of the new version to replace `vendor-test.js`. Ultimately, once the Populations.js library is updated to support WebPack builds directly, these workarounds should be eliminated.
+
+## Asset Attributions
+
+[Spider Image](https://openclipart.org/detail/73135/spider) from [openclipart](https://openclipart.org/) ([Creative Commons Zero 1.0 Public Domain License](http://creativecommons.org/publicdomain/zero/1.0/)).
+
+[Copy Icon](https://www.flaticon.com/free-icon/copy_832334#term=copy&page=1&position=77) by [SmashIcons](https://www.flaticon.com/authors/smashicons) from [FlatIcon](https://www.flaticon.com/) ([Creative Commons BY 3.0](http://creativecommons.org/licenses/by/3.0/) license).
+
+
+# create-react-app README 
```

Comment:
Are we required to include this Readme in our main Readme? Can we put it in a separate file and hyperlink? Or perhaps we can just link to the main create-react-app Readme?

Reply:
Done -- it's clearly an improvement. Thanks for the nudge.